src/SFA.DAS.ProviderApprenticeshipsService.Web/Controllers/CommitmentController.cs

```diff
@@ -93,28 +93,11 @@ public ActionResult DraftList(long providerId)
         }
 
         [HttpGet]
+        [Deprecated]
         [Route("{hashedCommitmentId}/Details", Name = "CohortDetails")]
-        public async Task<ActionResult> Details(long providerId, string hashedCommitmentId)
+        public ActionResult Details(long providerId, string hashedCommitmentId)
         {
-            LogUserClaims();
-
-            var model = await _commitmentOrchestrator.GetCommitmentDetails(providerId, hashedCommitmentId);
-               
-            model.BackLinkUrl = GetReturnToListUrl(providerId);
-             
-            AddFlashMessageToViewModel(model);
-
-            return View(model);
-        }
-
-        private void LogUserClaims()
-        {
-            var claims = ((ClaimsIdentity)HttpContext.User.Identity).Claims
-                        .Select(x => $"{x.Type}: {x.Value}").ToArray();
-
-            var logValue = string.Join(Environment.NewLine, claims);
-
-            _logger.Trace("User claims", new Dictionary<string, object> { { "providerClaims", logValue } });
+            return Redirect(_providerUrlhelper.ProviderCommitmentsLink($"{providerId}/unapproved/{hashedCommitmentId}/details"));
```

Comment:
should we be logging before these re-directs similar das-employercommitments?

Reply:
Updated to log before redirection